components/intl.rst

```diff
@@ -269,7 +270,45 @@ which defaults to the current default locale::
     $timezone = Timezones::getName('Africa/Nairobi', 'de');
     // => 'Ostafrikanische Zeit (Nairobi)'
 
-You can also check if a given timezone ID is valid::
+You can also get all the timezones that exist in a given country. The
+``forCountryCode()`` method returns one or more timezone IDs, which you can
+translate into any locale with the ``getName()`` method shown earlier::
+
+    // unlike locale codes, country codes are always uppercase (CL = Chile)
+    $timezones = Timezones::forCountryCode('CL');
+    // => ['America/Punta_Arenas', 'America/Santiago', 'Pacific/Easter']
+
+The reverse lookup is also possible thanks to the ``getCountryCode()`` method,
+which returns the code of the country where the given timezone ID belongs to::
+
+    $countryCode = Timezones::getCountryCode('America/Vancouver')
+    // => $countryCode = 'CA' (CA = Canada)
+
+The `UTC/GMT time offsets`_ of all timezones are provided by ``getRawOffset()``
+(which returns an integer representing the offset in seconds) and
+``getGmtOffset()`` (which returns a string representation of the offset to
+display it to users)::
+
+    $offset = Timezones::getRawOffset('Etc/UTC');              // $offset = 0
+    $offset = Timezones::getRawOffset('America/Buenos_Aires'); // $offset = -10800
+    $offset = Timezones::getRawOffset('Asia/Katmandu');        // $offset = 20700
+
+    $offset = Timezones::getGmtOffset('Etc/UTC');              // $offset = 'GMT+00:00'
+    $offset = Timezones::getGmtOffset('America/Buenos_Aires'); // $offset = 'GMT-03:00'
+    $offset = Timezones::getGmtOffset('Asia/Katmandu');        // $offset = 'GMT+05:45'
+
+The timezone offset can vary in time because of the `daylight saving time (DST)`_
+practice. By default these methods use the ``time()`` PHP function to get the
+current timezone offset value, but you can pass a timestamp as their second
+arguments to get the offset at any given point in time::
+
+    // In 2019, the DST period in Madrid (Spain) went from March 31 to October 27
+    $offset = Timezones::getRawOffset('Europe/Madrid', strtotime('March 31, 2019'));   // $offset = 3600
+    $offset = Timezones::getRawOffset('Europe/Madrid', strtotime('April 1, 2019'));    // $offset = 7200
+    $offset = Timezones::getGmtOffset('Europe/Madrid', strtotime('October 27, 2019')); // $offset = 'GMT+02:00'
+    $offset = Timezones::getGmtOffset('Europe/Madrid', strtotime('October 28, 2019')); // $offset = 'GMT+01:00'
+
```

Comment:
it might be worth documenting the 3rd arg as well :) to clarify the formatted GMT offset can differ depending on locale

https://github.com/symfony/symfony/blob/19811b88740c87534d95af1ed177075c2a5c01ce/src/Symfony/Component/Intl/Tests/TimezonesTest.php#L580-L583 (note how the time formatting `H` vs `HH` can also differ)